engine/core/src/main/java/org/pentaho/reporting/engine/classic/core/modules/output/table/html/helper/HtmlTextExtractorHelper.java

```diff
@@ -496,7 +503,8 @@ private boolean tryHandleRotatedText( final RotatedTextDrawable rotatedTextDrawa
   }
 
   private boolean tryHandleDrawable( final ReportAttributeMap attrs, final long width, final long height,
```

Comment:
javadoc HtmlTextExtractorHelper.java#tryHandleDrawable. More so explain what the new argument `boolean convertToBase64`  does. A developer shouldn't have to dig through the code to get a high level explanation of what is going on. **You are not responsible for the existing arguments.**